README.md

```diff
@@ -1,17 +1,34 @@
-# anti\_JSON_FASTA
+## Build and visualize trees from antibody JSON data. ##
 
-Build and visualize trees from antibody JSON data.
 
+
+First:
+	Need to activate the virtual environment to run the pipeline.
+	Copy and paste these commands:
+		- if you have Mac:
+		`conda create --name bcell --file spec-file.txt`
```

Comment:
Where is `spec-file.txt`? There is an `env.txt` below... is this what you were referring to?